ApplicationLibCode/Application/Tools/RiaEclipseUnitTools.cpp

```diff
@@ -62,20 +49,20 @@ double RiaEclipseUnitTools::darcysConstant( UnitSystem unitSystem )
 //--------------------------------------------------------------------------------------------------
 ///
 //--------------------------------------------------------------------------------------------------
-RiaDefines::DepthUnitType RiaEclipseUnitTools::depthUnit( UnitSystem unit )
+RiaDefines::DepthUnitType RiaEclipseUnitTools::depthUnit( RiaDefines::UnitSystem unit )
```

Comment:
This function makes it clear we should name the enum to EclipseUnitSystem

Se updated comment in #7127